test/event-stream/event-message-chunker-stream.spec.js

```diff
@@ -0,0 +1,95 @@
+var EventMessageChunkerStream = require('../../lib/event-stream/event-message-chunker-stream').EventMessageChunkerStream;
+var testEventMessages = require('./test-event-messages.fixture');
+var MockEventMessageSource = require('./mock-event-message-source-stream.fixture').MockEventMessageSource;
+
+describe('EventMessageChunkerStream', function() {
+    it('splits payloads into individual messages', function(done) {
```

Comment:
if we are not supporting Stream V1, you probably need to run this test only in stream v2. Like this:
```javascript
if (AWS.HttpClient.streamsApiVersion === 1) return;
```